### PR TITLE
Linebar rendering components addition

### PR DESCRIPTION
**Pull request by dylangiles**

### CODE REVIEW

File: src/Components/MainNav/MainNav.jsx [hunk 1]
```diff
@@ -40,6 +40,8 @@ class MainNav extends Component {
     render() {
         const menuitems = {
             system: [
+                {id: "salesproj", value: "Sales Projections"},
```
Comment (dylangiles) [inline src/Components/MainNav/MainNav.jsx]: New menu item for when we finish Sales Projection wizard

File: src/Components/MainNav/MainNav.jsx [hunk 2]
```diff
@@ -80,22 +82,22 @@ class MainNav extends Component {
                         <NavDropdown title="System" id="main-system">
                             {menuitems.system.map(
                                 (x, i) => x.id === 'divider' 
-                                ? (<NavDropdown.Divider/>)
-                                : <NavDropdown.Item id={x.id} onClick={this.handleClick}>{x.value}</NavDropdown.Item>
+                                ? (<NavDropdown.Divider key={i}/>)
+                                : <NavDropdown.Item key={i} id={x.id} onClick={this.handleClick}>{x.value}</NavDropdown.Item>
```
Comment (dylangiles) [inline src/Components/MainNav/MainNav.jsx]: Added keys to fix stupid React key complaints

File: src/Components/Rosters/EditRosterPage/EditRosterDayPane.jsx [hunk 3]
```diff
@@ -32,7 +34,7 @@ class EditRosterDayPane extends Component {
         const handleClose = () => this.setState({showModal: false});
         const handleShow = () => this.setState({showModal:true});
         const areas = [
-            "production", "service"
+            "managers Roster","production", "service"
```
Comment (dylangiles) [inline src/Components/Rosters/EditRosterPage/EditRosterDayPane.jsx]: Fix this at some point, no spaces in data names

**Comment by doqetsupaqld** [comment]
PR denied

**Pull request closed without merging**